gui-wm/river/river-0.3.2.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 2021-2022 Aisha Tammy
+# Distributed under the terms of the ISC License
+
+EAPI=8
+
+inherit edo
+
+DESCRIPTION="Dynamic tiling wayland compositor"
+HOMEPAGE="https://codeberg.org/river/river"
+
+SRC_URI="
+https://codeberg.org/river/river/releases/download/v${PV}/${P}.tar.gz
+https://codeberg.org/ifreund/zig-pixman/archive/v0.1.0.tar.gz -> zig-pixman-0.1.0.tar.gz
+https://codeberg.org/ifreund/zig-wayland/archive/v0.1.0.tar.gz -> zig-wayland-0.1.0.tar.gz
+https://codeberg.org/ifreund/zig-wlroots/archive/v0.17.0.tar.gz -> zig-wlroots-0.17.0.tar.gz
+https://codeberg.org/ifreund/zig-xkbcommon/archive/v0.1.0.tar.gz -> zig-xkbcommon-0.1.0.tar.gz
```

Comment:
nit: Maybe indent to match style for the rest of the file?

Reply:
Oh my, what happened here for so many tarballs?
Did their project structure change?

Reply:
@epsilon-0 No, in the past the author provided a tarball with all Git submodules included. But since Zig 0.12, the project adopted the native package manager from the language, which would fetch those tarballs.

Reply:
For reference, here is where those dependencies are defined: https://codeberg.org/river/river/src/branch/master/build.zig.zon